cocos/core/animation/skeletal-animation-blending.ts

```diff
@@ -63,11 +63,19 @@ export class BlendStateBuffer {
                 s = scale.value;
                 anyChanged = true;
             }
+
+            // Note: rotation and eulerAngles can not co-exist.
```

Comment:
cc @YunHsiao

Reply:
Should be more specific on this, it is the two curves that shouldn't coexist, not the properties per se.
And we should add in-editor restrictions on this @gameall3d